docs/core/idioms/pause-for-approval.md

```diff
@@ -0,0 +1,53 @@
+# Pause for Approval
+
+A manual-only trigger that has been applied to a task will pause the task once it is reached. The task can only be resumed by an individual or event, either from the Prefect UI or GraphQL API. 
+
+This example demonstrates the functionality by running the workflow with Prefect Cloud, using a cloud hook for a notification to approve the task run and then continuing with the approve button.
+
+Cloud hook to notify of flows in a paused state.
+
+![Cloud Hook Paused State](/idioms/pause_resume_cloud_hook.png)
+ 
+```python
+
+from prefect import task, Flow
+from prefect.triggers import manual_only
+
+@task
+def build():
+    print("build task")
+    return True
+
+@task
+def test(build_result):
+    print("test task")
+    build_result == True
+
+@task(trigger=manual_only)
+def deploy():
+    print("deploy task")
```

Comment:
I suggest a docstring on this task that reinforces that it will only run if someone approves it

Reply:
added